docs/plugins.md

```diff
@@ -64,13 +101,46 @@ The module must have a `package.json` file with a `pluginInfo` entry. The option
   }
 ```
 
+The `loadedBy` option tells Kuzzle to install and load the plugin only by corresponding instance types.  
+The accepted values are: `all`, `server` and `worker`.
```

Comment:
Is "loadedBy" now required, or do we consider that it has a default value to "all" if not set ?